populate/src/main/resources/seed/portals/demo/participants/jtsalk.json

```diff
@@ -0,0 +1,20 @@
+{
+  "participantUser": {
+    "username": "jtsalk@test.com",
+    "lastLoginHoursAgo": 5
+  },
+  "profile": {
+    "givenName": "Jonathan",
```

Comment:
Can we give him a first name that doesn't share 3 letters with Jonas, and an email that's a bit more visually distinguishable? I typically log in with chrome autocompleting the username, and I could easily see myself mixing up these logins

Reply:
Apparently he has another kid named Darrell!